packages/tailwind/src/kz-spacing.ts

```diff
@@ -0,0 +1,34 @@
+import { defaultTheme } from "@kaizen/design-tokens"
+
+export const kzSpacing = {
+  "spacing-2": ".125rem",
+  "spacing-4": ".25rem",
+  "spacing-8": ".5rem",
+  "spacing-12": ".75rem",
+  "spacing-16": "1rem",
+  "spacing-24": "1.5rem",
+  "spacing-32": "2rem",
+  "spacing-40": "2.5rem",
+  "spacing-48": "3rem",
+  "spacing-56": "3.5rem",
+  "spacing-64": "4rem",
+  "spacing-72": "4.5rem",
+  "spacing-80": "5rem",
+  "spacing-96": "6rem",
+  "spacing-112": "7rem",
+  "spacing-128": "8rem",
+  "spacing-160": "10rem",
+  "spacing-200": "12.5rem",
+  "spacing-240": "15rem",
+  "spacing-280": "17.5rem",
+  "spacing-320": "20rem",
+  "spacing-xs": defaultTheme.spacing.xs,
+  "spacing-sm": defaultTheme.spacing.sm,
+  "spacing-md": defaultTheme.spacing.md,
+  "spacing-lg": defaultTheme.spacing.lg,
+  "spacing-xl": defaultTheme.spacing.xl,
+  "spacing-xxl": defaultTheme.spacing.xxl,
+  "spacing-xxxl": defaultTheme.spacing.xxxl,
+  "spacing-xxxxl": defaultTheme.spacing.xxxxl,
+  "spacing-xxxxxl": defaultTheme.spacing.xxxxxl,
```

Comment:
Ohhh are we still keeping these? I thought we were replacing our tshirt tokens with the new ones

Reply:
While I'd like to remove it, I'm at two minds for it 🧠 🧠  If someone uses this but has a design that says spacing-xl will that cause confusion if it's not there?

My assumption is that spacing tokens will eventually be rolled out to the entire platform in figma but maybe leave this here as a stop gap?

Reply:
> has a design that says spacing-xl will that cause confusion if it's not there

I don't there's any references to spacing tokens in Figma, if that's what you mean. So we'll be fine there.

I'd be strongly in favour of removing these. We can create a map from old token to new token if we want, but even then I kinda doubt that would end up being useful to people.

Reply:
We're gonna have to bite the bullet on this sooner or later, and now is gonna be the best time to do it.

Reply:
We've removed these now 🥳